src/components/views/settings/tabs/room/RolesRoomSettingsTab.tsx

```diff
@@ -456,10 +456,10 @@ export default class RolesRoomSettingsTab extends React.Component<IProps> {
                     return null;
                 }
 
-                let label = plEventsToLabels[eventType];
-                if (label) {
+                let label: string;
+                if (plEventsToLabels[eventType]) {
                     const brand = SdkConfig.get("element_call").brand ?? DEFAULTS.element_call.brand;
-                    label = _t(label, { brand });
+                    label = _t(plEventsToLabels[eventType]!, { brand });
```

Comment:
it would have been preferable to rename `label` than to introduce an extra lookup in `plEventsToLabels` and the `!` operator, but never mind

Reply:
Sure